includes/admin/plugins.php

```diff
@@ -193,27 +193,45 @@ function perflab_render_plugin_card( array $plugin_data ) {
 		);
 	}
 
-	$details_link = esc_url_raw(
-		add_query_arg(
-			array(
-				'tab'       => 'plugin-information',
-				'plugin'    => $plugin_data['slug'],
-				'TB_iframe' => 'true',
-				'width'     => 600,
-				'height'    => 550,
-			),
-			admin_url( 'plugin-install.php' )
-		)
-	);
+	$title_link_attr = '';
+	if ( isset( $plugin_data['slug'] ) && current_user_can( 'install_plugins' ) ) {
+		$title_link_attr = ' class="thickbox open-plugin-details-modal"';
+		$details_link    = esc_url_raw(
+			add_query_arg(
+				array(
+					'tab'       => 'plugin-information',
+					'plugin'    => $plugin_data['slug'],
+					'TB_iframe' => 'true',
+					'width'     => 600,
+					'height'    => 550,
+				),
+				admin_url( 'plugin-install.php' )
+			)
+		);
 
-	$action_links[] = sprintf(
-		'<a href="%s" class="thickbox open-plugin-details-modal" aria-label="%s" data-title="%s">%s</a>',
-		esc_url( $details_link ),
-		/* translators: %s: Plugin name and version. */
-		esc_attr( sprintf( __( 'More information about %s', 'default' ), $name ) ),
-		esc_attr( $name ),
-		esc_html__( 'Learn more', 'performance-lab' )
-	);
+		$action_links[] = sprintf(
+			'<a href="%s" class="thickbox open-plugin-details-modal" aria-label="%s" data-title="%s">%s</a>',
+			esc_url( $details_link ),
+			/* translators: %s: Plugin name and version. */
+			esc_attr( sprintf( __( 'More information about %s', 'default' ), $name ) ),
+			esc_attr( $name ),
+			esc_html__( 'Learn more', 'performance-lab' )
+		);
+	} elseif ( ! empty( $plugin_data['homepage'] ) ) {
+		$title_link_attr = ' target="_blank"';
+
+		/* translators: %s: Plugin name. */
+		$aria_label = sprintf( __( 'Visit plugin site for %s', 'default' ), $name );
+
+		$details_link = $plugin_data['homepage'];
```

Comment:
I just noticed there is no `else` condition for if `$plugin_data['homepage']` is blank. In that case, it seems `$details_link` will never be defined and the reference below will be an error. PhpStorm's static analysis is warning me about this:

![image](https://github.com/WordPress/performance/assets/134745/0575d980-c698-4020-b90c-e4f668065fcf)

Is `homepage` ever blank? If so, I think we can fall back to using the WordPress.org plugin directory page in this case (which is always the same as the `homepage` in our plugins anyway):

```suggestion
	} else {
		$title_link_attr = ' target="_blank"';

		/* translators: %s: Plugin name. */
		$aria_label = sprintf( __( 'Visit plugin site for %s', 'default' ), $name );

		$details_link = ! empty( $plugin_data['homepage'] ) ? $plugin_data['homepage'] : __( 'https://wordpress.org/plugins/', 'default' ) . $plugin_data['slug'];
```

See related logic in core: https://github.com/WordPress/wordpress-develop/blob/80096ddf29d3ffa4d5654f5f788df7f598b48756/src/wp-admin/includes/plugin-install.php#L714-L717

Reply:
I'm confused by what `homepage` refers to. I guess it is always the WordPress.org plugin directory page? I guess the `Plugin URI` from the bootstrap file is not even returned in the API response?

Reply:
Oh no, the `homepage` _is_ the `Plugin URI`. In our case, this is not good because it points to the plugin directory on the GitHub repo. Therefore, this doesn't seem right. I've updated my suggestion above to not use `$plugin_data['homepage']` at all.

Reply:
I just realized my suggestion lacked the trailing slash from https://github.com/WordPress/wordpress-develop/blob/364ea85faca123a56a3631594ec7667164b1482a/src/wp-admin/includes/plugin-install.php#L715

Added in cc43ccf